README.md

```diff
@@ -492,9 +492,46 @@ For this example, we've changed the module `:my_module` to `:my_module2`, and th
   {"get-started", "quickstart"}
 ]}.
 ```
-
 <!-- tabs-close -->
 
+## Customizing Autocomplete
```

Comment:
Let's move this to the `mix docs` task. That's where we generally put options and configuration. WDYT?

Reply:
That’s fine with me but should I do that for the redirects section above? I put it in the resume because that’s where that was (Also something I added so i could just be wrong x 2 😅)

Reply:
Yes, for sure. I should have caught the redirect one the first time around as well. Please move both of them there and follow the style there!

Reply:
Done :)

The `redirects` option was already documented in the task, but I moved its accompanying long form docs there as well.